README.md

```diff
@@ -49,9 +49,40 @@ _tick_ is used for many industrial applications including:
 
 ## Installation
 
-### Requirements
+### Windows
 
-_tick_ currently works only on Linux/OSX systems and requires Python 3.4 or newer. Please have the required Python dependencies in your Python environment:
+  Requries:
+- python 3.4+
+- swigwin 3.0+
+- Microsoft Visual C++ (https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017)
+
+HowTO:
+
```

Comment:
perhaps we can just link to this https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx